frontend_v2/src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts

```diff
@@ -130,6 +130,12 @@ export class ChallengeviewallsubmissionsComponent implements OnInit, AfterViewIn
    */
   fieldsToGetExport: any = [];
 
+  /**
+   *Check whether team name is filtered
+   */
+  isTeamFiltered: boolean = true;
```

Comment:
@gautamjajoo the logic is not clear to me. Is this flag used to check if the query matches what we passed as parameter to filter API?

Reply:
So, basically the pagination number is taken from the URL and when we filter the team name, the URL changes as well, hence we use this flag to check if there is a filter added then we change our URL to a different one.